.releaserc.yaml

```
@@ -0,0 +1,33 @@
+# A semantic-release configuration for an Obsidian plugin
+# By using `npx`, we avoid the need to install `semantic-release` 
+#  and `semantic-release-obsidian-plugin` globally.
+# This configuration is set up to run in dry-run mode by default and 
+# uses the standard branches for Obsidian plugins (`master`, `main`, `beta` etc.)
+# run semantic-release as follows to release a new version of your plugin:
+#   dry-run (default):  npx -p github:brianrodri/semantic-release-obsidian-plugin semantic-release 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)